From 6e6877d601dc3a3bf047724c86fa18b1609a2057 Mon Sep 17 00:00:00 2001
From: Harini Katakam <harini.katakam@xilinx.com>
Date: Mon, 17 Mar 2014 13:16:48 +0530
Subject: [PATCH 090/509] SPI: cadence: Add short description in
 cdns_transfer_one_message

https://github.com/analogdevicesinc/linux.git xcomm_zynq
commit 05bd6503a5783dc0a71a09e2c035d767e704fe29

Add short description for function to fix kernel-doc warning.

Signed-off-by: Harini Katakam <harinik@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 drivers/spi/spi-cadence.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/spi/spi-cadence.c b/drivers/spi/spi-cadence.c
index 528018d..48093af 100644
--- a/drivers/spi/spi-cadence.c
+++ b/drivers/spi/spi-cadence.c
@@ -472,7 +472,7 @@ static int cdns_prepare_transfer_hardware(struct spi_master *master)
 }
 
 /**
- * cdns_transfer_one_message -
+ * cdns_transfer_one_message - Sets up and transfer a message.
  * @master:	Pointer to the spi_master structure which provides
  *		information about the controller.
  * @msg:	Pointer to the spi_message which contains the
-- 
1.7.5.4

